Testimonials/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/bootstrap.css">
+	<link href="https://fonts.googleapis.com/css?family=Lobster+Two" rel="stylesheet">
+	<title>Testimonials</title>
+	<script type="text/javascript" src="js/jquery.js"></script>
+	<script type="text/javascript" src="js/bootstrap.js"></script>
+</head>
+<style type="text/css">
+
+@font-face {
+	font-family: 'Product Sans';
+	src: url('fonts/Product Sans.ttf');
+}
+	
+.testimonials{
+	padding: 75px 0 150px 0;
+	background: #f8ffff;
+}
+
+.right-testifier-block{
+	padding-right: 50px;
+	padding-top: 35px;
+}
+
+.left-testifier-block{
+	padding-top: 35px;
+}
+
+.testifier-name{
+	font-family: 'Lobster Two', cursive;
+	font-size: 42px;
+}
+
+.testifier-designation{
+	font-family: 'Product Sans', sans-serif;
+	font-size: 16px;
+}
+
+.testifier-words{
+	font-family: 'Product Sans', sans-serif;
+	font-size: 18px;
+}
+
+</style>
+<body>
+<div class="testimonials">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-5 col-md-offset-1">
+				<img class="img-responsive img-circle" src="img/img1.jpg" width="350" height="350" />
+			</div>
+			<div class="col-md-5 right-testifier-block">
+				<h3 class="testifier-name">Dewansh Rawat</h3>
```

Comment:
Make final changes before submitting a PR. 
For instance, remove 'Dewansh Rawat' from here and write the final thing which needs to be displayed on the wesite